enterprise/utils.py

```diff
@@ -930,7 +930,10 @@ def get_last_course_run_end_date(course_runs):
     """
     latest_end_date = None
     if course_runs:
-        latest_end_date = max(course_run['end'] for course_run in course_runs if course_run['end'] is not None)
+        try:
+            latest_end_date = max(course_run['end'] for course_run in course_runs if course_run['end'] is not None)
```

Comment:
Do we need to check for isinstance(course_run['end'], datetime.date)?

Reply:
I think all dates are saved in the same format but let me confirm that.

Reply:
I see the discovery api is showing all dates in the same format.